integration/teleterm_test.go

```diff
@@ -743,6 +749,130 @@ func testWaitForConnectMyComputerNodeJoin(t *testing.T, pack *dbhelpers.Database
 	require.NoError(t, <-waitForNodeJoinErr)
 }
 
+func testDeleteConnectMyComputerNode(t *testing.T, pack *dbhelpers.DatabasePack) {
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+	t.Cleanup(cancel)
+
+	authServer := pack.Root.Cluster.Process.GetAuthServer()
+	uuid := uuid.NewString()
+	userName := fmt.Sprintf("user-cmc-%s", uuid)
+
+	// Prepare a role with rules required to call DeleteConnectMyComputerNode.
+	ruleWithAllowRules, err := types.NewRole(fmt.Sprintf("cmc-allow-rules-%v", uuid),
+		types.RoleSpecV6{
+			Allow: types.RoleConditions{
+				Rules: []types.Rule{
+					types.NewRule(types.KindNode, services.RW()),
+				},
+			},
+		})
+	require.NoError(t, err)
+	userRoles := []types.Role{ruleWithAllowRules}
+
+	_, err = auth.CreateUser(authServer, userName, userRoles...)
```

Comment:
This is not necessary, along with creating a new role with necessary rules. AFAIR if you use the creds generated in `TestTeleterm`, you should have admin-like access to cluster.

The only reason I created a new user and role and so on in `testCreateConnectMyComputerRole` is that `CreateConnectMyComputerRole` modifies the role list of the user. I wanted to run the tests in parallel, so I had to create a new user for each tests so that they wouldn't overwrite each other.

Here we're creating and then deleting a single node, so it shouldn't be necessary to create a new user for that.

Reply:
Unfortunately, the default user doesn't have permissions to remove nodes:
```
access denied to perform action "delete" on "node"
```